### PR TITLE
Vet Clinic Database - Create table

### PR DESCRIPTION
**Pull request by virag-ky**
In this project I created a database called, vet_clinic. In this database, I created a table called, animals. Throughout this project, I used PostgreSQL commands to create/add/query data.
Each row has a unique identifier that belongs to one animal.
I attach a [screenshot](https://user-images.githubusercontent.com/79658534/185114773-5f84b32b-795e-49d8-a304-b44ad23a35fe.png) of the results of the queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2018-12-31';
```
Comment (addod19) [inline queries.sql:4]: - [ ] It would be better if we can list the name of all animals born between 2016 and 2019.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name FROM animals WHERE neutered AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
```
Comment (addod19) [inline queries.sql:10]: - [ ] It would be better if the weight to be queried is between 10.4 and 17.3

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @virag-ky, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Nice job designing the schema and adding data into the database :100: 
- [ ] Queries are well implemented and supported with screenshots :+1: 
- [ ] Git-flow followed :rocket: 
- [ ] README is very descriptive :1st_place_medal: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (virag-ky) [reply queries.sql:4]: Screenshot from 2022-08-17 15-54-27

I'm not sure what is wrong here.

For me, this format works for the dates: '2016-01-01'. It is listing all the animal names between 2016 and 2019.

**Review by gemmen29 — APPROVED** [review]
Hi @virag-ky, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**